athenz/assertion.go

```diff
@@ -45,16 +53,9 @@ func expandPolicyAssertions(dn string, configured []interface{}) []*zms.Assertio
 	assertions := make([]*zms.Assertion, 0, len(configured))
 	for _, aRaw := range configured {
 		data := aRaw.(map[string]interface{})
-		role := data["role"].(string)
-		if !strings.Contains(role, ROLE_SEPARATOR) {
-			role = dn + ROLE_SEPARATOR + role
-		}
+		role := dn + ROLE_SEPARATOR + data["role"].(string)
```

Comment:
do we validate here something like `validateRoleNameWithinAssertion` ?

Reply:
We have already validated this within scheme